modules/custom/az_core/az_core.module

```diff
@@ -24,6 +25,31 @@ function az_core_help($route_name, RouteMatchInterface $route_match) {
   }
 }
 
+/**
+ * Implements hook_toolbar().
+ */
+function az_core_toolbar() {
+  $items = [];
+
+  $items['az_quickstart'] = [
+    'type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'link',
+      '#title' => t('Quickstart 2.0'),
+      '#url' => Url::fromRoute('az_core.az_quickstart'),
+      '#attributes' => [
+        'title' => t('AZ Quickstart 2.0'),
```

Comment:
```suggestion
        'title' => t('AZ Quickstart 2'),
```

Reply:
I think that making this change is enough to satisfy this ticket and make a good addition to Quickstart without spending a lot of additional effort on a low priority feature.